site/content/release-notes.md

```diff
@@ -4,13 +4,19 @@ title = "Release Notes"
 
 ## 0.8 (not released yet)
 
+### New Features
+
+* The already specified new message exchange pattern - called *one-way commands* - is now available to *business applications*. Therefore the `CommandClient` class was extended by a method `sendOneWayCommand` that does not expect a response from the device. See the `HonoExampleApplicationBase` class for how to use this new command pattern.
+  On the adapter side, this pattern is supported for HTTP and MQTT so far.
```

Comment:
why not AMQP?

Reply:
Not implemented yet - no technical reasons.
Do you think it is being essential for 0.8 Release?

Reply:
There is nothing to do for the AMQP adapter because it simply forwards the original message, i.e. if the original message does not contain a reply-to, then it is a one-way command. My point is that you can safely mention the AMQP adapter also supporting one-way commands :-)

Reply:
You are right, very good :+1: (haven't thought about that until now).
I will adapt the release notes.